mitxgraders/__init__.py

```diff
@@ -25,7 +25,7 @@
 from mitxgraders.listgrader import *
 from mitxgraders.formulagrader import *
 from mitxgraders.sampling import *
-from mitxgraders.baseclasses import ConfigError, InvalidInput
+from mitxgraders.exceptions import ConfigError, InvalidInput
```

Comment:
`InvalidInput` was here purely as a convenience to authors. I think this should now be `StudentFacingError`.

Reply:
That said, I like the idea of having an exceptions module. :+1: